src/com/parse4cn1/operation/IncrementFieldOperation.java

```diff
@@ -48,7 +48,6 @@ public Object apply(Object oldValue, ParseObject parseObject, String key)
             throws ParseException {
 
         if (oldValue == null) {
-            needIncrement = false;
```

Comment:
Thanks for the clarification. I'll take a closer look at this one before merging (especially since the tests are failing). Perhaps that flag is not needed at all.

By the way, thanks for your active work on parse4cn1 ;)